src/components/panels/hours-panel-date-viewer.js

```diff
@@ -0,0 +1,586 @@
+                marginTop: '8px'
+              }}
+            >
+              {weeks.map((week, weekIndex) => {
+                const currentlySelectedWeek = week.some((day) => {
+                  const viewedWeekStart = new Date(new Date().setDate(
+                    new Date().getDate() + weekOffset * 7 - new Date().getDay()
+                  ));
+                  const weekStartDay = startOfDay(day);
+                  const viewedWeekStartDay = startOfDay(viewedWeekStart);
+                  return weekStartDay.getTime() === viewedWeekStartDay.getTime();
+                });
+                return (
+                  <a
+                    onClick={(event) => {
+                      return updateWeekOffset({ dispatch, event, week, weekOffset });
+                    }}
+                    onKeyDown={(event) => {
+                      return updateWeekOffset({ dispatch, event, week, weekOffset });
+                    }}
+                    tabIndex={0}
+                    key={weekIndex}
+                    aria-label={`${currentlySelectedWeek ? 'This is the currently selected week.' : 'View'} week ${weekIndex + 1} in ${month} from ${getFormattedDate(week[0])} to ${getFormattedDate(week[week.length - 1])}`}
+                    css={{
+                      border: currentlySelectedWeek
+                        ? `2px solid var(--color-maize-400)`
+                        : `2px solid rgba(0,0,0,0)`,
+                      display: 'grid',
+                      gap: '4px',
+                      gridTemplateColumns: 'repeat(7, 1fr)',
+                      textDecoration: 'none'
+                    }}
+                  >
+                    {week.map((day, dayIndex) => {
+                      return (
+                        <div
+                          key={dayIndex}
+                          css={{
+                            backgroundColor: getColorBasedOnDate(day),
+                            borderRadius: '4px',
+                            color: day.getMonth() === currentBrowseDate.getMonth() ? 'inherit' : '#999',
+                            padding: '10px',
+                            textAlign: 'center'
+                          }}
+                        >
+                          {day.getDate()}
+                        </div>
+                      );
+                    })}
+                  </a>
+                );
+              })}
+            </div>
+            <p css={{ color: `var(--color-neutral-300);` }}>
+              <span css={{ background: 'white',
+                border: `solid 1px var(--color-neutral-200)`,
+                borderRadius: '4px',
+                boxShadow: `0 1px 1px rgba(0, 0, 0, .2), 0 2px 0 0 rgba(255, 255, 255, .7) inset;`,
+                display: 'inline-block',
+                fontFamily: 'monospace',
+                fontSize: '0.85rem',
+                marginBottom: SPACING.S,
+                padding: `0 ${SPACING['2XS']}` }}
+              >
```

Comment:
I know that the `kbd` element is used in `src/components/site-search.js`. Might be a good opportunity to create a reusable component?